network-api/networkapi/wagtailpages/migrations/0009_delete_softwareproductpage.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 3.2.12 on 2022-03-23 21:34
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('wagtailredirects', '0007_add_autocreate_fields'),
+        ('wagtailforms', '0004_add_verbose_name_plural'),
+        ('wagtailinventory', '0002_pageblock_unique_constraint'),
+        ('wagtail_footnotes', '0004_footnote_translatable_mixin_migration'),
+        ('wagtailcore', '0066_collection_management_permissions'),
+        ('wagtailpages', '0008_research_hub_site_structure'),
```

Comment:
very curious why it decided to add all of these when the only dependency is `wagtailpages` O_o

Reply:
Should I remove the other ones?